e2e-tests/gef/cypress/fixtures/MOCKS/MOCK_DEALS.js

```diff
@@ -0,0 +1,217 @@
+const CONSTANTS = require('../constants');
+const { sub, getUnixTime } = require('date-fns');
+
+const today = new Date();
+const threeDaysAgo = sub(today, { days: 3 });
+const threeDaysAgoUnix = getUnixTime(threeDaysAgo).toString();
+
+const MOCK_AIN_APPLICATION = {
+  // _id: '61a7710b2ae62b0013dae687',
+  dealType: 'GEF',
+  // userId: '619bae3467cc7c002069fc1e',
```

Comment:
we can remove these properties if they are not being used

Reply:
done